var/spack/repos/builtin/packages/gmsh/package.py

```diff
@@ -23,6 +23,7 @@ class Gmsh(CMakePackage):
     license("GPL-2.0-or-later")
 
     version("master", branch="master")
+    version("4.13.1", sha256="d99ebcd50aef4e081fae53e6b3f84cadded4da0b6806a4fb2421d2d0beed2432")
```

Comment:
This sha256 appears to (now?) be `5a6db8c94176624d8a134fd0c7b9f5399bfe1965f2f35c8c3af6a5ab273bc827`.

Reply:
`v4.13.1` seems to be continuously changing?? Today I got a different hash than yours yesterday.

I've reverted the change and will wait until `v4.13.1` is out of active development.